ops/otel/README.md

```diff
@@ -0,0 +1,12 @@
+# OpenTelemetry Collector
+
+An OpenTelemetry Collector is used to aggregate tracing/metrics data and export it to a wide variety of configurable
+observability systems (jaeger, grafana, google cloud etc).
+
+Valkyrie sends its tracing/metric data in OpenTelemetry Protocol (OTLP) format over HTTP to the Collector.
+
+You can test running an otel-collector locally using:
+
+```bash
+docker-compose -f docker-compose-local.yaml up
+```
```

Comment:
Lets explain that the local setup includes Jaeger for visualization and add the URL for web interface for the impatient(lazy) reader 😏

Reply:
Good suggestion 😄